docs: create a guide to deploy nfts on RSK (#823)
rsksmart/devportal: guides/nft/index.md

---

> title: 'Introduction | Deploy an NFT project on the RSK'

Reviewer: ```suggestion
title: 'Introduction | Deploy an NFT project on the RSK Testnet'
```

Author: Updated the overview and index page as suggested...

---

> title: 'Deploy NFT on RSK'

Reviewer: ```suggestion
title: 'Introduction | Deploy an NFT project on the RSK'
```

layout: 'rsk'
---

<!-- guides/nft/index
 -->

Welcome to the Complete Guide on **How to Create and Deploy an NFT Smart Contract on RSK**.

### Introduction

In this tutorial, we will take you through a step by step process on how to write a ERC-721 smart contract, and deploy to the RSK testnet using the following tools:
Solidity, Hardhat, Ether.js, RSK Testnet, IPFS, Pinata, and Test RBTC from a faucet.
Not to worry, if these terms are new to you — we will explain them!

## Contents

- [Overview](/guides/nft/overview/)
- [What is a Non-fungible Token (NFT)](/guides/nft/overview/#what-is-a-non-fungible-token-nft/)
    - [What is ERC-721?](/guides/nft/overview/#what-is-erc-721/)
    - [What is Hardhat?](/guides/nft/overview/#what-is-hardhat/)
    - [What is Ethers.Js?](/guides/nft/overview/#what-is-ethersjs/)
    - [What is IPFS?](/guides/nft/overview/#what-is-ipfs/)
- [Configure the metadata for your NFT using Pinata](/guides/nft/configure-nft-metadata/#configure-the-metadata-for-your-nft-using-pinata/)
    - [Hardhat Configuration](/guides/nft/configure-nft-metadata/#hardhat-configuration/)
- [Create your Smart Contract](/guides/nft/create-smart-contract/#top/)

> - [How to deploy your NFT to RSK](/guides/nft/deploy-nft-on-rsk/#how-to-deploy-your-nft-to-rsk/)

Reviewer: ```suggestion
- [How to deploy your NFT to RSK](/guides/nft/deploy-nft-on-rsk/#how-to-deploy-your-nft-on-rsk/)
```
